python/oneflow/mock_torch/__init__.py

```diff
@@ -217,6 +217,16 @@ def __call__(self, *args, **kwargs):
 
 
 class enable:
+    """
+    mock_torch.enable(lazy=None,verbose=None) -> None
+
+    Enable mock function.
+
+    Args:
+        lazy (bool, optional): When lazy=True, a false object is returned for a non-existent interface without an immediate error.
```

Comment:
```suggestion
        lazy (bool, optional): When lazy=True, a dummy object is returned for a non-existent interface without an immediate error.
```

我英语水平不是很好，dummy object 似乎在程序员术语里更常见些